scripts/releaseUtils.ts

```diff
@@ -100,7 +100,7 @@ export function getVersionChoices(currentVersion: string): VersionChoice[] {
   const isStable = !currentBeta && !currentAlpha;
 
   function inc(i: ReleaseType, tag = currentAlpha ? "alpha" : "beta") {
-    return semver.inc(currentVersion, i, tag)!;
+    return semver.inc(currentVersion, i, tag) || "0";
```

Comment:
what was the reason for this change?

We definitely want to throw an error if this lib doesn't return a result we'd expect.

Reply:
lint warning - updated to throw an error